yaml/com/ibm/License/LicenseManager.interface.yaml

```diff
@@ -0,0 +1,41 @@
+description: >
+      This Method accepts license key and forwards to host
+properties:
+     - name: LicenseString
+       type: string
+       description: >
+         License string of 34 characters.
+     - name: LicenseActivationStatus
+       type: enum[self.Status]
+       default: Pending
+       description: >
+         License Activation status
```

Comment:
This is the last uploaded License? 
I.e. we upload a license this says "Activated" for the next x mins? Until a reboot?

Reply:
Yes. This is the status of the license upload action